generic/account_supplier_billing/models/supplier_billing.py

```diff
@@ -105,3 +121,10 @@ def action_cancel(self):
                     'supplier_billing_id': False,
                 })
         return res
+
+    @api.multi
+    def _compute_billing_amount(self):
```

Comment:
What is the purpose of this field?

Reply:
i will use this fields to calculate amount - wht but now i don't use this fields so i will delete it.

Reply:
OK